IdentityCore/tests/MSIDKeyOperationUtilTest.m

```diff
@@ -0,0 +1,251 @@
+//
+//  MSIDKeyOperationUtilTest.m
+//  IdentityCore
+//
+//  Created by Ameya Patil on 9/29/21.
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+#if !TARGET_OS_SIMULATOR
+#import <XCTest/XCTest.h>
+#import "MSIDKeyOperationUtil.h"
+#import "MSIDJwtAlgorithm.h"
+#import "NSData+MSIDTestUtil.h"
+#import "NSData+JWT.h"
+#import "NSData+MSIDExtensions.h"
+
+@interface MSIDKeyOperationUtilTest : XCTestCase
+    @property (nonatomic) SecKeyRef eccPrivateKey;
+    @property (nonatomic) SecKeyRef eccPublicKey;
+    @property (nonatomic) SecKeyRef rsaPrivateKey;
+    @property (nonatomic) SecKeyRef rsaPublicKey;
+    @property (nonatomic,nullable) NSString *sharedAccessGroup;
```

Comment:
is nullable needed?

Reply:
I don't think so, I'm not sure why I added it. Removed it.